department-of-veterans-affairs/va.gov-cms
# VACMS-14205 Fix profile required fields. #14026

## Description

Closes #14205 

relates to #12187

## Testing done


## Screenshots


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As content editor edgar.cruse 
1. visit https://pr14233-hghtp6e3u8qmoo00kebzk48sppmv1isn.ci.cms.va.gov/tuscaloosa-health-care/staff-profiles/james-hardemon
2. Edit the profile
3. Check the box for "Create profile page with biography"
4. delete any values in the First sentence and Body text
5. **UNCheck** the box for "Create profile page with biography"
6. Save the page with a revision log message
   - [x] Validate that it saves without error.
7. Edit the node.
8. Check the box for "Create profile page with biography"
9. Save the page with a revision log message
   - [x] Validate save is prevented with a notificationpointing to "First sentence" field (HTML 5 validation happens before save even occurs)
10. Add content to the "First sentence" field
11. Save the page with a revision log message
   - [x] Validate save is prevented with a notification pointing to "First sentence" field  (This is prevented after a true save attempt, not HTML5 pre-save)  **Note:** The jump link from the error report at the top of the page does not jump to the error.  This is a core bug and is not specific to this issue.
12. Add content to the "Body Text" field
13. Save the page with a revision log message
   - [x] Validate that it saves correctly.
14.  Edit the node.
15. **UNCheck** the box for "Create profile page with biography"
16. Save the page with a revision log message
   - [x] Validate that it saves correctly.



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
